nixos/modules/services/web-apps/keycloak.nix

```diff
@@ -672,8 +660,8 @@ in
               mkdir -p /run/keycloak/ssl
               cp $CREDENTIALS_DIRECTORY/ssl_{cert,key} /run/keycloak/ssl/
             '' + ''
-              export KEYCLOAK_ADMIN=admin
-              export KEYCLOAK_ADMIN_PASSWORD=${escapeShellArg cfg.initialAdminPassword}
+              export KC_BOOTSTRAP_ADMIN_USERNAME=admin
```

Comment:
I _think_ this option should now be nullable, and default to `null` that allows a temporary admin user? (not super strong opinion tho)

Reply:
Done